RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,322 @@
+| [Docker Scan](https://build.ci.opensearch.org/job/docker-scan/)                                                               |     Workflow that checks vulnerabilities for a given docker image as an input.                |
+
+
+#### Release Workflows
+
+| Wokflow                                                                                  | Description         |
+| ---------------------------------------------------------------------------------------- | ------------------- |
+| [Release Notes Tracker](https://build.ci.opensearch.org/job/release-notes-tracker/)      | Workflow that identifies if a component has a release notes added based on the commit history.  |
+| [Promote Repos](https://build.ci.opensearch.org/job/distribution-promote-repos/)         |   Workflow that signs and promotes the APT/YUM repos to the production buckets accessed via the cloudfront.                 |
+| [Promote artifacts](https://build.ci.opensearch.org/job/distribution-promote-artifacts/) | Workflow that signs and promotes all the release artifcats           to the production buckets accessed via the cloudfront.         |
+| [Publish to Maven](https://build.ci.opensearch.org/job/publish-to-maven/)                |   Workflow that signs and publishes to the central maven repository.                  |
+| [Docker Promotion](https://build.ci.opensearch.org/job/docker-promotion/)                |  Workflow that promotoes the docker images to production docker repositories.                   |
+| [Validation Workflow](https://build.ci.opensearch.org/job/distribution-validation)       |     Workflow that validates the released distribution.                 |
+
+
+#### Creating a New Version
+
+Each new OpenSearch release process starts when any one component increments a version, typically on the `main` branch. For example, [OpenSearch#1192](https://github.com/opensearch-project/OpenSearch/pull/1192) incremented the version to 2.0. The [version check automation workflow](.github/workflows/versions.yml) will notice this change or it can be triggered [manually](https://github.com/opensearch-project/opensearch-build/actions/workflows/versions.yml), and make a pull request (e.g. [opensearch-build#514](https://github.com/opensearch-project/opensearch-build/pull/514)) that adds a new manifest (e.g. [opensearch-2.0.0.yml](manifests/2.0.0/opensearch-2.0.0.yml). After that's merged, a GitHub issue is automatically opened by [this workflow](.github/workflows/releases.yml) to make a new release using [this release template](.github/ISSUE_TEMPLATE/release_template.md) (e.g. [opensearch-build#566](https://github.com/opensearch-project/opensearch-build/issues/566)). Existing and new components [(re)onboard into every release](ONBOARDING.md) by submitting pull requests to each version's manifest.
+
+### Release Manager
```

Comment:
This is confusing. `Creating a New Version` should happen after a Release Manager volunteers, but in this process doc we mention "Creating a New Version" before obtaining a RM volunteer. IMHO, this process outlined here should be in order to make it clear. Perhaps with a simple ordered list (e.g., 1. Release Manager, 2. Feature freeze, 3....)

Reply:
@nknize This is a standalone release guide but now the flow chart. This will be finally linked to the [release template ](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) that will act as a step by step SOP for the release manager.